### ISSUE TITLE
Add missing migration guides for 0.15

### ISSUE THREAD

**Issue by alice-i-cecile** [bevyengine/bevy-website#1831]
> https://github.com/bevyengine/bevy/pull/14082 still seems missing. 

 _Originally posted by @rparrett in [#1724](https://github.com/bevyengine/bevy-website/issues/1724#issuecomment-2491185955)_

To check manually:

- [x] https://github.com/bevyengine/bevy/pull/6042: non-breaking
- [x] https://github.com/bevyengine/bevy/pull/14917: non-breaking
- [ ] https://github.com/bevyengine/bevy/pull/15090: missing
- [x] bevy_reflect: Add `Reflectable` trait sha: 69541462c5ef70914fd719fcd1f503045c76a453: non-breaking
- [ ]  https://github.com/bevyengine/bevy/pull/14082: missing
- [x] The Cooler 'Retain Rendering World' sha: 56f8e526dde49b4e4ad62efb7ad27bfe0bd6f617

**Comment by rparrett** [comment]
There are likely other totally missing sections -- I think that this one and perhaps a few others are missing due to the way we do filtering by date in the generation tool. Some PRs that were breaking and created during the 0.14 RC period would fall into the date range of `0.14.0..whatever`.

**Comment by alice-i-cecile** [comment]
Fixing #1518 and re-running the tool is probably the least painful way to resolve this.

**Comment by alice-i-cecile** [comment]
Even after removing the filter on time closed, I'm still getting the following output.
```
PR not found for Allow mix of hdr and non-hdr cameras to same render target sha: 027f8e21ecedec4844ff0e93c077d166ce259891
PR not found for rename bevy_state_macros to bevy_state_macros_official sha: 401234a5fb6865ea9ddd5740d47e13d022959fef
PR not found for Update crate metadata for bevy state sha: 98bc5ff8c4731e8b8fec5babc815c8111173a4b5
PR not found for Document the Release Candidate process sha: ecdd1624f302c5f71aaed95b0984cbbecf8880b7
PR not found for rename the crate bevy_state_macros_official back to its original name sha: b17292f9d11cf3d3fb4a2fb3e3324fb80afd8c88
PR not found for improved the error message by insert_boxed (issue #13646) (again) sha: d45bcfd0434702c0b97de1727e5d5a9b2055dad3
PR not found for Make the component types of the new animation players clonable. sha: 651f3d08d75077c4b12f519d3b22ecc047a9670f
PR not found for Uncouple `DynamicTextureAtlasBuilder` from assets sha: d38d8a148a65074ef6620046b2e9349105412fdd
PR not found for Add `mappings` to `EntityMapper` sha: 3bfc42766621d12749c09eca3bfd9911dc0dc3f5
PR not found for 13743 app exit hang sha: 7b14b8c01531c8a9997d5bcf3ff2ca40a5f26857
PR not found for Clear messed up feature flag on AppExitStates impl sha: e9a3a0d694e04e5090a2c03ff312e654733434a7
PR not found for view.inverse_clip_from_world should be world_from_clip sha: b9e67bc3c3f466d2f1e86b27cd6a937086aa0fdb
PR not found for Fix EntityCommands::despawn docs sha: 7c9c6ff27f483d592a3591be4c80f7e0316496e5
PR not found for Bump crate-ci/typos from 1.21.0 to 1.22.3 sha: 3478d158ccbe79818a17b228e77c2859a47c1202
PR not found for Let init_non_send_resource require FromWorld instead of Default sha: 69d7da9b37b98a8e9044bedf8d9886b718ea310c
PR not found for Fix minor doc typo sha: 1b9edd0e5baf31fde5333e08fe3882b558d12830
PR not found for 2D top-down camera example sha: 33dff0d3f7fd9f68d9b30b83b2763c39e7a980f1
PR not found for Add segments to `ExtrusionBuilder` sha: 1196186257e785e268bb61f2759c88e9d1be3664
PR not found for Stable interpolation and smooth following sha: a569b35c188f9bd79213c5efd2ae34fa97c8aaa7
PR not found for Improve error handling for `AssetServer::add_async` sha: 0dfdd87248702fd691c7348d750565981e6d7807
PR not found for Remove unused mip_bias parameter from apply_normal_mapping sha: c50a4d8821c16fe0ba678223b2d1fb5af73c761f
PR not found for Adds back in way to convert color to u8 array, implemented for the two RGB color types, also renames Color::linear to Color::to_linear. sha: 298b01f10da4535ff70b026f960a9eca5b6901f0
PR not found for Meshlet misc sha: 50ee483665963023f93009220b7053c5b572f675
PR not found for Add more granular system sets for state transition schedule ordering sha: 6d0b7504a25c6d38c2230d7f312b768f3a39eb39
PR not found for Clarify error message due to missing shader file sha: 2356276afd78a523f3511f8b389fba5fd79855e7
PR not found for fix docs around `StateTransition` and remove references to `apply_sta… sha: 7ec301c48d90e59f47eead9dee84ab16a776cf77
PR not found for Make FOG_ENABLED a shader_def instead of material flag sha: 5134272dc9f416cbda83b23a74ad367066a6c93b
PR not found for Re-name and Extend Run Conditions API sha: 70a38ab1f69a810b60074fc290c7278c21596061
PR not found for Update serialize flag for bevy_ecs sha: 93f3432400deab424901c43512e7c174814fa7b1
PR not found for Added an illustration to the compass direction docs (issue 13664) sha: 2c5959a29da817dcc78d2e8c5790922863c2cf7f
PR not found for Remove extra call to clear_trackers sha: 49661b99fe2f02e2d66e8af6d3b3584eb5ca7a26
PR not found for Poll system information in separate tasks sha: f187d9c5fc771f6da863ed2810bb83c2e225a9c2
PR not found for Fix meshlet vertex attribute interpolation sha: fd30e0c67dbea269fa2f266c3ad71ca86bfbf048
PR not found for Revert Align Scene::write_to_world_with to match DynamicScene::write_to_world_with sha: 54010cc07e7ece41e925cd13b5748e42e8ff39c2
PR not found for Custom primitives example sha: c172c3c4b556109a04714fe5ed8bdc39759365a5
PR not found for Add `from_color` to `StandardMaterial` and `ColorMaterial` sha: d803adff0941019f728e07ee86cb4d147c427119
PR not found for Split event.rs into a full module. sha: 2ccdae74896756dde535011388498b918f9c27d4
PR not found for Revert "Make FOG_ENABLED a shader_def instead of material flag (#13783)" sha: d659a1f7d506c5c6eba9dfe2a6e878a8c72ecef6
PR not found for Revert "constrain WorldQuery::init_state argument to ComponentInitial… sha: 07a85676b3051a41c545c8ed0296449865f55591
PR not found for Fix minor typos in query join docs sha: f23c686fcf4a4d57cfd551aa66b5b82ee17c652b
PR not found for Highlight dependency on shader files in examples sha: 435d9bc02cff8b1cd762d0c9d75e20157b3238bb
PR not found for Ensure that events are updated even when using a bare-bones Bevy App sha: 2cffd14923c9e217dc98881af6c8926fe4dc65ea
PR not found for run windows ci on rust 1.78 sha: 14fdec674dd64fdd3bb00b25d67db72cff4a1d75
PR not found for Add missing plugins to doc of DefaultPlugins sha: ba198151a4b8ffd216b0cfbcf220d6439e58d882
PR not found for Use dynamic uniform buffer in post processing example sha: 061baa4ff7d1ceb6bb2d4c74b9798d8cc8bf5e37
PR not found for reduce the antialias strength sha: 004ba585b2e2dd915bb9b9d6972ec9c80c8cfed2
PR not found for Wgpu 0.20 sha: 2825ac8a8e69b4c816e3ca67df2f841904ac7b69
PR not found for Mention updating Bevy book code validation for release candidates sha: 117346d8b789a2ce95a8f32436ccda62dd3a43a9
PR not found for fix non-exact text h-alignment sha: 01971f210e37be7779353db7efc5814699c9a808
PR not found for Fix is_plugin_added::<Self>() being true during build sha: 1395e3672cd262e51e1b6e3c5b9d12b02cb90998
PR not found for Restore overwrite capabilities of `insert_state` sha: 1a1b22ede8a183df440afb62d35d03264dc6ce18
PR not found for Generalised ECS reactivity with Observers sha: eb3c81374a703bb5630869819b382a5a1d5a2e90
PR not found for Upstream `CorePlugin` from `bevy_mod_picking` sha: aaccbe88aa0d591c9c741f690ab472785c7bac09
PR not found for text position: use size instead of bounds sha: d8f42608f94d432f8a50bd0f29abbfbf7220edce
PR not found for Allow bevy_color use without bevy_reflect support sha: 8b25ef33283dcef79f67c55b3eb494042334701c
PR not found for remove inaccurate warning from `in_state` sha: f69117331b8b88a711674931e5095ca44830cedd
PR not found for Warn about missing `StatesPlugin` when installing states sha: 90894a1910897a309f9e3d88b0466be2c4bb2562
PR not found for Make time_system public sha: cde610577d5a9f9544dd31693f378faa040ba9bf
PR not found for observers example doesn't follow standards sha: 8b38299bd46f5a582c61131be011fd6fd96d9f49
PR not found for Bump crate-ci/typos from 1.22.3 to 1.22.7 sha: 13f9b6993ddd501d061892fa85383aa5b06ed391
PR not found for Fix phantom key presses in winit on focus change (#13299) sha: 92ac77867d0db41adfaffab40bce7a7d07751ee5
PR not found for Use a unstable sort to sort component ids in `bevy_ecs` sha: 16e02e18891a90a9991ea786032a48b375267516
PR not found for Add first person view model example sha: 48f70789f58357742379096fabb5c651f56c312b
PR not found for Made some things in bevy_render `Debug`. sha: c75610e2b24bfd45bb200c22a8bf9a4a0773e34e
PR not found for Update accesskit and accesskit_winit sha: 8626ad05bc712de42b7399e52876e7eef2e2dd4f
PR not found for Update observer archetype flags for sparse components sha: 335dcf96a20f31e8145e7b8fe83a103a3817d4fa
PR not found for Fix parameter name of all_tuples's document sha: 47f58ac6c5b398c580f766d04f36ec415340c53c
PR not found for Fix minor typo sha: d1dd61477bb0092bd0fd4dc4ac75523b1ebda211
PR not found for Add cross gizmos sha: 4b3246af4082da39a3d400a268ce81694b1a7937
PR not found for Fix lints introduced in Rust beta 1.80 sha: 6273227e09053d916824bd378b8f9a4c3979da26
PR not found for bevy_reflect: Improve reflection serialization error messages sha: 53910e07aefc40a9f9ca089f643d731b748c699f
PR not found for Reuse VisibleEntities in check_light_mesh_visibilty sha: 41ad4e98de62d247bba8f8f0fec92303ba3a2d4f
PR not found for Refactor check_light_mesh_visibility for performance #1 sha: 91cd84fea7fa582c5ed029cc3ffe675b1e1bc6c1
PR not found for Improve MeshletMesh::from_mesh performance sha: 001cc147c671eb314958caa4fcc5d87fbe080730
PR not found for Specify units in `AmbientLight::brightness` docs sha: c37e81b34ae0aa4b118fe80e6d9241be987c7dad
PR not found for Use smooth_nudge in 2d_top_down_camera example sha: 20eb13f4588245550937611f72fb5e608fd27178
PR not found for Fix typo in `Query::single_mut` docs sha: 1b0475f2345f32c23579c808a898ad70821b0f19
PR not found for `IntoSystemConfigs::chain_ignore_deferred`'s return type fix sha: 6b2d4834e9afda8dffaf418958534b7082df83b4
PR not found for Shader code paths sha: f8014e0d01e9a48a94e5f7e5e38f33f4c873653b
PR not found for Fixed a link to Blender's mikktspace docs sha: 0a003eaa98f24130a5a198c7129fd92abeb8dc2d
PR not found for Improve error handling for log filter sha: 45a5f66c9ddbb9566fcf8b5d2c586af9768e72ae
PR not found for Use a well defined type for sides in RegularPolygon sha: 524dce750590f3654b526f239615f5775d86276a
PR not found for Use u32 for all resolution/subdivision fields in bevy_gizmos sha: eddb006a8a2612bb042d02e3f90a42fa2583ccba
PR not found for bug: Fix 9-slice textures with asymmetric borders. sha: 6df0e9b37d8db2990ccf92895b4bdba4ff492b3f
PR not found for docs(bevy_state): fix broken links in init_state and insert_state sha: 5bab161386784cb2505b3fac22c1f5bff0e3e454
PR not found for Fix typo in `ComponentId` docs: `of` -> `or` sha: e34ecf2f8669947cb5858337dc169c3a115c1ea3
PR not found for Change World::inspect_entity to return an Iterator instead of Vec sha: ee2487a6e2b91d6df3cb8ecf98cc71e19965a87d
PR not found for Move `StateTransitionSteps` registration to states plugin sha: 31af7249442962069da3fccc053ff9de7a18c402
PR not found for Use a ship in Transform::align example sha: 4d3f43131e441e7c6b24e34fff3f09804a9c2dfd
PR not found for Use u32 for resolution/subdivision in primitive meshing sha: 38c8dc27c7fd92649e27869c4c0df1e8c077a616
PR not found for Make meshlet processing deterministic sha: 6eec73a9a50eacbd63dee0afea206931e622a1cf
PR not found for feat: add insert_after and insert_startup_before sha: 2f9c42bb339deffa9a254244cc7c3bedf7b427f6
PR not found for Correctly check physical size when updating winit sha: 8635a0f916f314d109bfcd27cf5ba03bcf5b51fb
PR not found for Updated descriptions for some geometric primitives to include more detail sha: dcb191bb1837027156584260c3999558dd6368c0
PR not found for Use default window and font sizes in sprite_slice example sha: c110b103679cda647fabe19d455d5c73b195d447
PR not found for Omit font size where it closely matches the default in examples sha: c66c2c7420bf99b8406314f8670c4e393c6ba33f
PR not found for Fix a few "repeated word" typos sha: e46e246581240ad8ec4133f1276e7991616ac877
PR not found for Fix a couple typos in contributing docs sha: 68db79862227885bb5d4794c2636e1749e36e574
PR not found for bug: Don't panic. Warn on missing file_watcher path. (new branch) sha: e72f4ef9e902eb84f528de3c0461c19f4b3e0c25
PR not found for apply window scale to window size when creating it sha: 841df150ccd08e2b60bec61ddc1ba5983427ffd1
PR not found for Make Observer::with_event (and other variants) unsafe sha: 0daa6c510b1cc16d60b7df03be8c661456a06a55
PR not found for Fix meshlet interactions with regular shading passes sha: 158ccc6d6a45eb048cc3f40388f9cff0fc633375
PR not found for Bump crate-ci/typos from 1.22.7 to 1.22.9 sha: c500228227de396cc039094e45db689e5d5a8dac
PR not found for fix panic: invalid SlotMap key used sha: 87fa69bd7e71457edec4d77422b712c202a72503
PR not found for Add a test asserting that reflected cloning a Handle increments strong count sha: 3a82d6784bcbedf051ff9544fd870d3f77ed33f5
PR not found for Have WindowPosition::Centered take scale_factor_override into account sha: d48ebdf6416ea4f693ae3a3d1bb9b615a35f28b9
PR not found for Document use of `NO_COLOR` in `LogPlugin` sha: 82fd3df719b5e8d400d079a7a94f28ea47c4442d
PR not found for improved error message when forgetting to call system apply function … sha: 2d30ae437c31d77c14c05cd25a285e2dcd38f53c
PR not found for Use CameraController in 3d_gizmos example sha: 9cb9969793d446f3146ea3e894d11a782e513bc3
PR not found for Add example enum Component usage to ecs_guide sha: 6ec1f3e6f83a03367956000b0dd788d30cc554b7
PR not found for Document trace_tracy_memory in profiling.md sha: 54f1c4e21f1cb0c3fee5d0ae253d234cdb5acd9a
PR not found for Add Display implementation to DebugName. sha: 1df811e6a563b59dfb7f5f5eea52d03ec78fb6dd
PR not found for feat(bevy_app): expose an API to perform updates for a specific sub-app. sha: f51a306b30dfde0a3e315e291686ce70d4ace365
PR not found for Mark events as read during `EventReader::par_read` sha: 4c3b4a445dc2987eef75a151743943b878f93b52
PR not found for Don't show `.to_bits` in `Display` impl for `Entity` sha: 52e5ad5da7b59b2771cdec7b3637caa8a5c8a97e
PR not found for Fix MeshletMesh material system ordering sha: d8b45ca1369875bd066841ecd7f66f2d5b21e79e
PR not found for Remove unnecessary compute for rotation interpolation sha: dbffb41e504765078ca8a21f2bdaa8a3f97e6c50
PR not found for Make default behavior for `BackgroundColor` and `BorderColor` more intuitive sha: 336fddb1019cad526398e0ccdb28b9b0317635bb
PR not found for AnyOf soundness fix sha: 8308ad08a2d0903343368bece912878920d79a4b
PR not found for don't crash without features `bevy_pbr`, `ktx2`, `zstd` sha: 19d078c60953858d5cb7e006c5d90110712280a4
PR not found for Print warning when using llvmpipe sha: 0ae7afbcad07c4c01a14a99f4ad653eb2c07e90a
PR not found for  Fair Change Detection Benchmarking sha: f0bdce7425cdf4bb744ff8ed91a367d64c254dfa
PR not found for Use an opaque type for `EntityCommand::with_entity` sha: a4c621a127bebac0c6d8e870521f4b351b0ab7b5
PR not found for Refactor check_light_mesh_visibility for performance #2 sha: a3f91a28fc4c3a73836a93e35b464a3212cdd756
PR not found for example showcase: keep the order of the shaders imported sha: f757fe80b229afc6a43c12258bc9b0a240b7368c
PR not found for fix examples color_grading and mobile after BackgroundColor changes sha: ee63cf45c6efbf8a364949184e8e6a5a94ae3dc0
PR not found for Emit a warning if the result of `EntityCommand::with_entity` is not used sha: 2b7d54b3004c8bbedd116bf5e7ade349b8220ef0
PR not found for impl `BuildChildrenTransformExt` for `EntityWorldMut` sha: 57ac8f5211fca31d311a81fe1b1712ed1b51e8d3
PR not found for Fix typo in CPU adapter warning sha: 0ee9827ba246ac8d9f21a21c3c4ca4404e8ff59f
PR not found for only run one update per frame drawn sha: 3a04d38832fc5ed31526c83b0a19a52faf2063bd
PR not found for Improve MeshletMesh::from_mesh performance further sha: 4cd188568aa2e8b75e061e21614f179ec07392dd
PR not found for Windows CI example runner: back to using rust stable sha: 63fd8fbae5921042006cc3c81b750821ef3adec9
PR not found for Fix incorrect computation of mips for cluster occlusion lookup sha: 9148847589fe140159e862a40f20056ed03efc10
PR not found for add missing sort_unstable_by_key to QueryIter sha: e813326c877a6d871286c82c2f0addbf9eb482e5
PR not found for Add missing StaticSystemParam::queue implementation. sha: 1baa1a11b7f46f4c156a0d3aa72c3c592c35dcda
PR not found for Allow phase items not associated with meshes to be binned. sha: 44db8b7facd700c20213bc9fa42558a2bfd74901
PR not found for add PartialEq to Outline sha: edca8707c8501e3807281ff1b267e96933adf5e8
PR not found for Fix  error in AnyOf sha: 6573887d5c5f57142373e0009c1a78b9c5e10bd5
PR not found for Update ui_material example to be a slider instead sha: 011f71a24520d1401b1d890a0c088c4ae7d41fb6
PR not found for Made Material2dBindGroupId instantiable sha: e69e28fe74d496ada15ee47e3515b498ef8f74ba
PR not found for don't put previous skin/morph in the morphed_skinned_mesh_layout sha: 8a7d3ce461058bad2e0aaa608353abe2cd4da6d0
PR not found for Fix compile failure in WASM without `wgpu` backend sha: a1545dd3a6aa8a7a3ae29666af2018ba87a24b77
PR not found for Support operations for render layers and fix equality comparisons sha: 64bc81181515d0988a5f1aca333e68bd2132daba
PR not found for Make gLTF node children Handle instead of objects sha: cb4fe4ea9e13636b4194362808f2af0b99482fd9
PR not found for Handle `Ctrl+C` in the terminal properly sha: f607be8777e3706c9023b6c5a3877565770189e0
PR not found for Mouse input accumulation sha: 6dcff2bfe8e209943e5ab67f90f0ff643de91785
PR not found for Merge `BuildWorldChildren` and `BuildChildren` traits. sha: ace4eaaf0e4201db0658f926d418d48de1a7821e
PR not found for Bump rodio to 0.19 sha: b389e7baba163feed478b1be45e8b194d6f0fbd2
PR not found for Bump `accesskit` to 0.16 sha: f7bc0a01dd4825f1b12602bb1052e182555d7ed6
PR not found for Clarify the difference between default render layers and `none` render layers sha: 9055fc1d68bc3ee0f965ad0d101b7e777a184e17
PR not found for Apply Clippy lints regarding lazy evaluation and closures sha: 856b39d821781e4b27206608cc93874a6e746d19
PR not found for deregister events sha: bd7dcd3f6ddd3cf7934c8c76bab2bad624b3e24c
PR not found for Optimize common usages of `AssetReader` sha: 5876352206d1bcea792825bf013eb212383b73d6
PR not found for Fix `push_children` inserting a `Children` component even when no children are supplied sha: 7aaf440fbfbd1e93c1cfc50130d5f3515e105d5c
PR not found for add missing mention of sort_unstable_by_key in QuerySortedIter docs sha: 1d907c5668a58da984ac98819ca80cf5e2eaa146
PR not found for Added `get_main_animation` for `AnimationTransitions` sha: ce5254e3ec3aa3f071c42b77b584679923a27125
PR not found for Fix footgun for Windows users in `fast_config.toml` sha: a58c4f7825010fe4c0e7332c8318176530c28e36
PR not found for use associated type bounds in QueryManyIter and QueryIter::sort() sha: 7e0d262d7728f545b89b04dc9820ddedbc293cd6
PR not found for Added feature switch to default Standard Material's new anisotropy texture to off sha: a47b91ccccd6af2802cab03c8e8c55196c2c5137
PR not found for fix remaining issues with background color in examples sha: 160bcc787c9b2f8dacafbf9dca7d7a6b2349386a
PR not found for Using simple approx round up in ui_layout_system sha: 2893fc3e8bdec79f9bf1c738767cacdc9a7c6843
PR not found for Fix border color in `ui_texture_slice` and `ui_texture_atlas_slice` examples. sha: c8d8ea6d4fb9140a256e13fe07006aa790f7ec25
PR not found for Skip extract UiImage When its texture is default sha: 1c2f6872023cb19d85f72ab707611768cdb0c19c
PR not found for Add `condition_changed` and `condition_became_true` to `common_conditions` sha: 12f005a024c163333455bfd8112893c6a8075716
PR not found for Depreciate `LoadAndSave` Asset Processor sha: dac4a5bbb4336527ffd0880233b92f5bc4f93c54
PR not found for The Cooler 'Retain Rendering World' sha: 56f8e526dde49b4e4ad62efb7ad27bfe0bd6f617
```/

**Comment by rparrett** [comment]
If I recall correctly, last time I looked, there was also some suspicious matching-by-PR-name that could have easily been replaced with a match-by-PR-number. I suspected that renamed PRs were also not being included.
